src/components/Good/Good.js

```diff
@@ -6,24 +6,39 @@ export class Good extends React.Component {
     isSelected: false,
   };
 
-  clickHandler = () => {
+  clickHandlerAdd = () => {
     this.setState(prevState => ({ isSelected: !prevState.isSelected }));
     this.props.setGood(this.props.word);
   };
 
+  clickHandlerRemove = () => {
+    const value = this.props.selectedGood.filter(
+      good => good !== this.props.word,
+    );
+
+    this.props.removeSelection(value);
+  }
+
   render() {
     return (
       <>
-        <li className={this.props.word === this.props.selectedGood
+        <li className={this.props.selectedGood.includes(this.props.word)
           ? 'selected' : null}
```

Comment:
```suggestion
          ? 'selected' : ''}
```
or use classnames library